README.md

```diff
@@ -73,7 +73,9 @@ rm -rf $HOME/.rustup/toolchains/stable-aarch64-apple-darwin
 mv $HOME/.rustup/toolchains/1.72.1-aarch64-apple-darwin $HOME/.rustup/toolchains/stable-aarch64-apple-darwin
 ```
 
-2. If you run into OOM errors, especially while running evaluations that use LLMs as evaluators like toxicity and
+2. Due to `fmeval`'s use of [Ray](https://www.ray.io/), and the fact that [Ray on Windows is still in beta](https://docs.ray.io/en/latest/ray-overview/installation.html#windows-support), users running evaluations on Windows machines may encounter Windows-specific issues. For example, `ray.init()` may raise an OS error, as described in [this thread](https://discuss.ray.io/t/ray-init-raises-oserror-assignprocesstojobobject-failed/7018).
```

Comment:
Question: This seems to be an important limitation that users should know about. Should we move this to the first line item under Troubleshooting?

Reply:
Good point. I was actually thinking of whether this deserves its own section, but I think I'll just put it at the top of Troubleshooting, like you've suggested.